src/github.com/getlantern/flashlight/pubsub/pubsub.go

```diff
@@ -0,0 +1,19 @@
+// package pubsub provides a global publish and subscribe infrastructure
+// for lantern
+package pubsub
+
```

Comment:
This is a wrapper around https://github.com/asaskevich/EventBus to make it easier to use at the expense of these being global.